spectralevents_ts2tfr.m

```diff
@@ -67,12 +67,13 @@
 sf = f/width;
 st = 1/(2*pi*sf);
 
-t=-3.5*st:dt:3.5*st;
+t=0:dt:3.5*st;
+t=[-t(end:-1:2),t];
 m = morlet(f,t,width);
 
 y = conv(s,m);
 y = 2*(dt*abs(y)).^2;
-y = y(ceil(length(m)/2):length(y)-floor(length(m)/2));
+y = y(floor(length(m)/2) + 1:length(y)-floor(length(m)/2));
```

Comment:
Checking - is this line for readability purposes/matching python code? I'm pretty sure I'm getting the same result whether I use ceil(length(m)/2) or floor(length(m)/2) + 1.

Reply:
Those two expressions are only equal when `length(m)` is odd. For example, when `length(m)` = `2`, `ceil(2/2)` = `1` whereas `floor(2/2) + 1` = `1`.